pkg/mesos/types.go

```diff
@@ -1,8 +1,17 @@
 package mesos
 
 import "strings"
+import "github.com/mesos/mesos-go/api/v1/lib/master"
 
-// Master represents a single mesos master node
+// Agents represents the JSON returned when using GET_AGENTS.
+type Agents struct {
```

Comment:
We could directly use this type : https://github.com/mesos/mesos-go/blob/f27edc01ca67f9b7e1b6a4ea9a1086e5372eaf0c/api/v1/lib/master/master.pb.go#L932